code/game/objects/items/bodybag.dm

```diff
@@ -51,13 +52,20 @@
 			name = "body bag - "
 			name += t
 			overlays += "bodybag_label"
+			tagged = 1
 		else
 			name = "body bag"
 		return
 	else if(istype(I, /obj/item/weapon/wirecutters))
-		user << "<span class='notice'>You cut the tag off of [src].</span>"
+		user << "<span class='notice'>You cut the tag off [src].</span>"
 		name = "body bag"
 		overlays.Cut()
+		tagged = 0
+
+/obj/structure/closet/body_bag/update_icon()
+	..()
+	if (tagged)
+		overlays += "bodybag_label"

```

Comment:
this is on purpose, the way the bodybag_label overlay is placed, it would show up on both opened and closed body bags. (basically, the open bodybag sprite opens 75% or so of the way; the remaining 25% is on the left side where the tag overlay is. this is hard to describe in words but it'll be very clear with the actual sprite.)